aries-site/src/examples/components/layer/LayerSideDrawerExample.js

```diff
@@ -0,0 +1,141 @@
+import React, { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Box,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  ResponsiveContext,
+  Select,
+  TextArea,
+  TextInput,
+  Text,
+} from 'grommet';
+import { Close } from 'grommet-icons';
+
+const LayerForm = ({ setOpen }) => {
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+    setOpen(false);
+  };
+
+  return (
+    <Box gap="medium">
+      <Box>
+        <Header align="start" pad={{ horizontal: 'xxsmall' }}>
+          <Box gap="xxsmall">
+            <Heading level={3} margin="none">
+              Form Header
+            </Heading>
+            <Text>a subtitle if needed</Text>
+          </Box>
+          <Button icon={<Close />} onClick={() => setOpen(false)} />
+        </Header>
+      </Box>
+      <Form
+        validate="blur"
+        onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+      >
+        <FormField
+          label="Text Area"
+          htmlFor="text-area-input"
+          name="textAreaInput"
+        >
+          <TextArea
+            id="text-area-input"
+            name="textAreaInput"
+            resize="vertical"
+          />
+        </FormField>
+        <FormField
+          label="Select Component"
+          htmlFor="select-one"
```

Comment:
Label is unattached. For Select inputs, need to modify like this:

```suggestion
          htmlFor="select-one__input"
```